wezterm-gui/src/overlay/copy.rs

```diff
@@ -708,9 +749,9 @@ impl CopyRenderable {
                         y: self.cursor.y,
                     };
                     self.start.replace(coord);
-                    self.select_to_cursor_pos();
                 }
                 self.selection_mode = *mode;
+                self.select_to_cursor_pos();
```

Comment:
note: this change is necessary to be able to immediately see selection mode changes (otherwise the change is only visible on next movement)